tests/executors/test_kubernetes_executor.py

```diff
@@ -674,20 +674,11 @@ class TestKubernetesExecutor(unittest.TestCase):
     def test_run_next_exception(self, mock_get_kube_client, mock_kubernetes_job_watcher):
 
         # When a quota is exceeded this is the ApiException we get
-        r = HTTPResponse()
-        r.body = {
-            "kind": "Status",
```

Comment:
this code is wrong, because of `urllib3/response.py:189`
```
def __init__(self, body='', headers=None, status=0, version=0, reason=None,
                 strict=0, preload_content=True, decode_content=True,
                 original_response=None, pool=None, connection=None, msg=None,
                 retries=None, enforce_content_length=False,
                 request_method=None, request_url=None):
...

        if body and isinstance(body, (basestring, bytes)):
            self._body = body
```

Reply:
https://github.com/urllib3/urllib3/blob/master/src/urllib3/response.py#L195-L234